### PR TITLE
[Hlm -2278]: add readme for services

### PR DESCRIPTION
**Pull request by Anushka-egov**

### CODE REVIEW

File: health-services/facility/README.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+
+d) POST `/facility/v1/bulk/_update` - Update bulk Facility, This API is internally call from facility controller to update the details of existing facilities in bulk.
+
+e) POST `/facility/v1/_delete` - Delete Facility, This API is internally call from facility controller to soft delete details of an existing facility.
+
+f) POST `/facility/v1/bulk/_delete` - Delete bulk Facility, This API is internally call from facility controller to soft delete details of an existing facility in bulk.
+
+g) POST `/facility/v1/_search` - Search Facility, This API is internally call from facility controller to search existing facility.
+
+### Property Dependencies
```
Comment (shivab-egov) [inline health-services/facility/README.md:39]: Remove Property Dependencies section if it is empty.

File: health-services/facility/README.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+https://editor.swagger.io/?url=https://raw.githubusercontent.com/egovernments/health-campaign-services/v1.0.0/docs/health-api-specs/contracts/registries/facility.yml#!/
+
+### Service Details
+
+#### API Details
+BasePath `/facility/v1`
+
+Facility service APIs - contains create, update, delete and search end point
+
+a) POST `/facility/v1/_create` - Create Facility, This API is internally call from facility controller to create/add a new facility.
```
Comment (shivab-egov) [inline health-services/facility/README.md:25]: Change this `Create Facility, This API is internally call from facility controller to create/add a new facility.` to `Create a single Facility using the provided request data.`

File: health-services/facility/README.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+
+b) POST `/facility/v1/bulk/_create` - Create bulk Facility, This API is internally call from facility controller to create/add new facilities in bulk.
```
Comment (shivab-egov) [inline health-services/facility/README.md:27]: Change this `Create bulk Facility, This API is internally call from facility controller to create/add new facilities in bulk.` to `Create multiple facilities using the provided request data.`

File: health-services/facility/README.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+
+c) POST `/facility/v1/_update` - Update Facility, This API is internally call from facility controller to update the details of an existing facility.
```
Comment (shivab-egov) [inline health-services/facility/README.md:29]: Update description, we are not making any internal API calls from controller.

**Review by shivab-egov — CHANGES_REQUESTED** [review]

File: health-services/facility/README.md (hunk 2 above)
Comment (Menka-egov) [reply health-services/facility/README.md]: Resolved

File: health-services/facility/README.md (hunk 4 above)
Comment (Menka-egov) [reply health-services/facility/README.md]: Resolved

**Review by roopeshegov — APPROVED** [review]

**Pull request merged**